### PR TITLE
spg chapter 17

### PR DESCRIPTION
**Pull request by SimonGross**
Also includes some weird changes as a result of removing windows line endings

### CODE REVIEW

File: chapter11/spg/Stereotypes.pm [hunk 1]
```diff
@@ -3,10 +3,12 @@ package Stereotypes;
 use strict;
 require Exporter;
```
Comment (utopia-2) [inline chapter11/spg/Stereotypes.pm:4]: This is no longer needed (though doesn't really hurt).

File: spg/My-List-Util/lib/My/List/Util.pm [hunk 2]
```diff
@@ -1,161 +1,164 @@
-package My::List::Util;
```
Comment (alexdeas) [inline spg/My-List-Util/lib/My/List/Util.pm:1]: Weird diffs, why does it think you rewrote the whole file?

Comment (SimonGross) [inline spg/My-List-Util/lib/My/List/Util.pm:1]: I was getting some windows line endings showing up in my diffs, so I tried to fix it. That just made  caused the whole thing to show up as a diff for some reason.

Comment (Spinodal) [inline spg/My-List-Util/lib/My/List/Util.pm:1]: adding `?w=1` to the end of the url sets the ignore whitespace flag in git!

**Pull request merged**